client/index.html

```diff
@@ -1,15 +1,22 @@
 <!doctype html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-    	<meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta name="mobile-web-app-capable" content="yes">
-        <link rel="icon" sizes="124x124" href="favicon.png">
-		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
-		<title>x-map</title>
-		<!-- Latest compiled and minified CSS -->
-	</head>
-	<body>
-		<div id="app"></div>
-	</body>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <link rel="icon" sizes="124x124" href="favicon.png" />
+    <title>X-Map</title>
+    <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous" /> -->
```

Comment:
alright. but with this base styles and the markup samples I've provided you should be able to fit any of the data in the layout, design improvements can be done secondly.

I'm good with what we have decided, keeping bootstrap for prototyping, just saying that the new given html/css is doing what bootstrap do, the same thing, but much simpler and totaly focused for our application (differently of bootstrap that is addressed for anything).